backend/models/job.js

```diff
@@ -1,69 +1,105 @@
+/**
+ * Schema for Job document
+ */
+
 import mongoose from "mongoose";
+
 const { Schema, model } = mongoose;
 
-const JobSchema = new Schema({
-    title: {
-        type: String, 
-        required: true,
-    },
-    clientName: { // TODO Confirm - Client Name should not be same as client logged in?
-        type: String,
-        required: true,
-    },
-    phoneNumber: { // TODO Confirm - Do we need a seperate phone or should it the number on record for the client?
-        type: String,
-        required: true,
-    },
-    client: {
-        type: mongoose.Types.ObjectId,
-        ref: "User",
-        required: true,
-    },
-    deliveryDate: {
-        type: String,
-        required: true,
-    },
-    pickupLocation: {
-        type: String,
-        required: true,
-    },
-    dropoffLocation: {
-        type: String,
-        required: true,
-    },
-    description: {
-        type: String,
-        required: true,
-    },
-    packageQuantity: {
-        type: Number,
-        required: true,
-    },
-    price: {
-        type: Number,
-        required: true,
-    },
-    applicants: [
-        {
+
+// Job Status
+export const JOB_STATUS_CREATED = 'CREATED';
+export const JOB_STATUS_ASSIGNED = 'ASSIGNED';
+export const JOB_STATUS_COMPLETED = 'COMPLETED';
+export const JOB_STATUS = [JOB_STATUS_CREATED, JOB_STATUS_ASSIGNED, JOB_STATUS_COMPLETED];
+
+// Stored for validation during update request
+export const FIELDS_OWNER_PERMITTED_TO_UPDATE = ['title', 'clientName', 'phoneNumber', 'deliveryDate', 'pickupLocation', 'dropoffLocation', 'description', 'packageQuantity', 'price'];
+export const OWNER_LIMITED_FIELDS = ['applicants', 'assignedDriver'];
```

Comment:
Yeah I was wondering where to place this and ended up placing it here so that we don't forget to update this list when we update any attributes in jobs API